experimental/traffic-portal/src/app/api/profile.service.spec.ts

```diff
@@ -116,6 +136,27 @@ describe("ProfileService", () => {
 		await expectAsync(responseP).toBeResolvedTo(profile);
 	});
 
+	it("sends request for Export object by Profile ID", async () => {
+		const profileId = 1;
+		const id = typeof(profileId) === "number" ? profile.id : profileId;
```

Comment:
The type of `profileId` is `const 1` so it will always be a `number`, so the branch using `profileId` will never be taken here. I'm not sure what you're trying to do, or I'd have something more helpful to say.